www/docs/mp/index.php

```diff
@@ -383,7 +383,7 @@
 
         $policiesList = new MySociety\TheyWorkForYou\Policies;
         $divisions = new MySociety\TheyWorkForYou\Divisions($MEMBER);
-        $policySummaries = $divisions->getMemberDivisionDetails();
+        $policySummaries = $divisions->getMemberDivisionDetails($strong_only = true);
```

Comment:
There's another use of getMemberDivisionDetails in the default section some lines down, but presumably that's no longer used, so could be removed in previous commit?